### PR TITLE
#22681 Improve docs for TLS options of remote console

### PR DESCRIPTION
**Pull request by kirill-da**
https://github.com/DACH-NY/canton/issues/22681
Note that for `2.10` and `3.x` we will fix the configuration to unify the config options, so this comment in the remote console docs is not required.

* [x] Document `transport-security` and `custom-trust-certificates.pem-file` for `sequencer/domain` nodes in 2.8 and 2.9

### CODE REVIEW

**Review by yves-da — APPROVED** [review]
LGTM, thanks!

Note: We're not updating the Canton docs artifacts because the config is spread to several config files which do not get resolved for a `literalinclude`. Also, since 2.8 and 2.9 are soon obsolete with the upcoming 2.10 release, spending effort creating updated Canton docs artifacts for this case does not make great sense.

File: docs/2.8.11/docs/canton/usermanual/console.rst [hunk 1]
```diff
@@ -85,7 +85,21 @@ TLS and Authorization
 ^^^^^^^^^^^^^^^^^^^^^
 
 For production use cases, in particular if the Admin Api is not just bound to localhost, we recommend to enable
-:ref:`TLS <tls-configuration>` with mutual authentication.
+:ref:`TLS <tls-configuration>` with mutual authentication. Note that for remote `domain` or `sequencer` nodes
+for `public-api` configuration uses different TLS settings than the `admin-api` configuration, i.e.:
```
Comment (carrielaben-da) [inline docs/2.8.11/docs/canton/usermanual/console.rst:89]:
```suggestion
:ref:`TLS <tls-configuration>` with mutual authentication. Note that for remote `domain` or `sequencer` nodes
the `public-api` configuration uses different TLS settings than the `admin-api` configuration:
```

File: docs/2.9.5/docs/canton/usermanual/console.rst [hunk 2]
```diff
@@ -85,7 +85,21 @@ TLS and Authorization
 ^^^^^^^^^^^^^^^^^^^^^
 
 For production use cases, in particular if the Admin API is not just bound to localhost, we recommend to enable
-:ref:`TLS <tls-configuration>` with mutual authentication.
+:ref:`TLS <tls-configuration>` with mutual authentication. Note that for remote `domain` or `sequencer` nodes
+for `public-api` configuration uses different TLS settings than the `admin-api` configuration, i.e.:
```
Comment (carrielaben-da) [inline docs/2.9.5/docs/canton/usermanual/console.rst:89]:
```suggestion
:ref:`TLS <tls-configuration>` with mutual authentication. Note that for remote `domain` or `sequencer` nodes
the `public-api` configuration uses different TLS settings than the `admin-api` configuration:
```

**Review by carrielaben-da — APPROVED** [review]
Small change.

**Pull request merged**